src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -666,6 +667,32 @@ private async Task<bool> EnumerateFilesFromArtifactsProvider(TContext globalCont
                     continue;
                 }
 
+                if (Path.GetExtension(filePath).IsArchiveByFileExtension())
+                {
+                    Stream stream = artifact.Stream;
+                    stream ??= new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+                    ZipArchive archive;
+
+                    try
+                    {
+                        archive = new ZipArchive(stream, ZipArchiveMode.Read, leaveOpen: true);
+                    }
+                    catch(InvalidDataException)
+                    {
+                        // TBD log corrupt zip file.
```

Comment:
>TBD [](http://example.com/codeflow?start=27&length=3)

I guess the same TBD applies in the other catch I just commented on. Can you at least comment there too?